NEWS.md

```diff
@@ -27,6 +28,141 @@ The format is based on [Keep a Changelog]
+  in the same way as each potential `environment` file.
+- Allow empty variables in `environment` files. In other words, respond to
+  variable declarations of the form "VARIABLE=", with no following value,
+  by setting the corresponding environemtn variable as an empty string.
+- Add optional headless fallback output that is automatically created when
+  no other output exists.  Enable this by setting the environment variable
+  `LABWC_FALLBACK_OUTPUT` to the desired output name.  The feature
+  benefits applications like wayvnc the most by ensuring that there is
+  always an output available to connect to.
+  Co-Authored-By: Simon Long <simon@raspberrypi.com>
+- Optionally show windows all on all workspaces in window-switcher.
+
+```xml
+<windowSwitcher allWorkspaces="yes">
+```
+
+- Handle touch on headerbar using cursor emulate events. Issue #1550
+  Written-by: @spl237
+- Updated dbus activation environment with more environment variables
+  (`XCURSOR_SIZE`, `XCURSOR_THEME`, `XDG_SESSION_TYPE`, `LABWC_PID`)
+  Written-by: @winerysearch  Issue #694
+- Run `shutdown` script on exit (equivalent to `autostart` on startup)
+- Add `wrap` argument to action `MoveToOutput`. Wrap is disabled by
+  default to keep the user interface consistent. Example usage:
+
+```xml
+<action name="MoveToOutput" direction="right" wrap="yes" />
+```
+
+### Fixed
+
+- Fix workspace-switcher on-screen-display positioning of text using
+  right-to-left (RTL) locales. Written-by: @micko01 Issue #1633
+- Unconstrain xdg-shell popups to usable area (rather than full output) so
+  that popups do not cover layer-shell clients such as panels.
+  Written-by: @tokyo4j
+- Exclude unfocusable XWayland windows (for example notifications and
+  floating toolbars) from being processed by wlr-foreign-toplevel protocol
+  processing as these windows should not be shown in taskbars/docks/etc.
+- Render text buffers with opaque backgrounds because subpixel text
+  rendering over a transparent background does not work properly with
+  cairo/pango. PR #1631
+- Fallback on layout 'us' if a keymap cannot be created for the provided
+  `XKB_DEFAULT_LAYOUT`. If keymap still cannot be created, exit with a
+  helpful message instead of a segv crash.
+- Reload cursor theme and size on reconfigure. Written-by: @spl237
+  Issue #1587
+- Fix a number of surface-focus releted short-comings:
+  - Handle cursor-button-press on layer-shell subsurfaces and fix bug in
+    `get_cursor_context()` which resulted in layer-surfaces not being
+    detected correctly. PR #1594
+  - Overhaul the logic for giving keyboard focus to layer-shell clients.
+    PR #1599
+- Fix move/resize bug manifesting itself on touchpad taps with
+  `<tapAndDrag>` disabled because libinput sends button press & release
+  signals so quickly that `interactive_finish()` is never called.
+  Written-by: @tokyo4j
+- Include always-on-top windows in window-switcher
+- Make resize flicker free when running labwc nested
```

Comment:
suggest adding `again`, e.g. `Make resize flicker free again when running labwc nested` (it was a regression of wlroots 0.17).

Reply:
Done.